subprojects/dependency-management/src/main/java/org/gradle/internal/component/model/LocalComponentDependencyMetadata.java

```diff
@@ -331,12 +403,14 @@ public TaskDependency getArtifactBuildDependencies() {
     }
 
     private static class Matcher {
```

Comment:
The private static classes in here have grown large enough that I think they deserve their own file.